package.json

```diff
@@ -6,15 +6,18 @@
     "url": "https://github.com/OfficeDev/Excel-Custom-Functions.git"
   },
   "license": "MIT",
+  "config": {
+    "app-type-to-debug": "desktop",
+    "dev-server-port": 3000,
+    "source-bundle-url-path": "index.win32" 
```

Comment:
where does this get read from? I don't see where we read in the config settings?

Reply:
https://github.com/OfficeDev/Office-Addin-Scripts/pull/39/files#diff-85bacdc64694e4466f9d30d7dc8096b9R58